charts/netdata/values.yaml

```diff
@@ -284,7 +284,7 @@ child:
       path: /etc/netdata/stream.conf
       data: |
         [stream]
-          enabled = yes
+          enabled = {{ ternary "yes" "no" .Values.parent.enabled }}
```

Comment:
I never used that kind of function in helm, tho I love it in terraform. I learned something new, thx :D

Reply:
Helm extends go template lang with [sprig functions](http://masterminds.github.io/sprig/).